getting-started/dc2-guide.rst

```diff
@@ -0,0 +1,182 @@
+########################################
+Processing DC2 data with the Gen3 Butler
+########################################
+
+The data explored in this guide is simulated data, the same kind used for Rubin's Data Preview 0 (DP0).
+
+.. note::
+
+   This guide assumes the user has access to a shared "Gen3" Butler repository containing data from the Dark Energy Science Collaboration (DESC)'s Data Challenge 2 (DC2), most likely on the ``lsstdevl`` machines at NCSA.
+   This guide further assumes the user has a recently-built version of :py:mod:`lsst.distrib` from the LSST Science Pipelines.
```

Comment:
Could you include the tag of the build you were using when you wrote this guide? E.g. `lsst-scipipe-0.6.0` with `w_2021_30`